[2023-10-18 09:29:49,367: INFO: main: welcome to custom log]
[2023-10-18 21:35:46,074: INFO: main: welcome to custom log]
[2023-10-19 08:54:29,241: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-19 08:54:29,354: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-19 08:54:29,404: INFO: common: created directory at: artifacts]
[2023-10-19 08:54:29,412: INFO: common: created directory at: artifacts/data_ingestion]
[2023-10-19 08:54:29,420: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2023-10-19 08:54:50,313: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2023-10-20 07:46:25,304: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-20 07:46:25,465: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-20 07:46:25,490: INFO: common: created directory at: artifacts]
[2023-10-20 07:46:25,559: INFO: common: created directory at: artifacts/data_ingestion]
[2023-10-20 07:46:25,562: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2023-10-20 07:47:03,793: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2023-10-20 07:55:48,720: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<< ]
[2023-10-20 07:55:48,752: INFO: common: yaml file: config\config.yaml loaded successfully ]
[2023-10-20 07:55:48,752: INFO: common: yaml file: params.yaml loaded successfully ]
[2023-10-20 07:55:48,752: INFO: common: created directory at path :artifacts ]
[2023-10-20 07:55:48,752: INFO: common: created directory at path :artifacts/data_ingestion ]
[2023-10-20 07:55:48,752: INFO: data_ingestion: file already exists ]
[2023-10-20 07:55:51,841: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x ]
[2023-10-20 07:56:38,454: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<< ]
[2023-10-20 07:56:38,456: INFO: common: yaml file: config\config.yaml loaded successfully ]
[2023-10-20 07:56:38,458: INFO: common: yaml file: params.yaml loaded successfully ]
[2023-10-20 07:56:38,459: INFO: common: created directory at path :artifacts ]
[2023-10-20 07:56:38,461: INFO: common: created directory at path :artifacts/data_ingestion ]
[2023-10-20 07:56:40,245: INFO: data_ingestion: artifacts/data_ingestion/data.zip download :
Content-Type: text/html; charset=utf-8
X-Robots-Tag: noindex, nofollow, nosnippet
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Pragma: no-cache
Expires: Mon, 01 Jan 1990 00:00:00 GMT
Date: Fri, 20 Oct 2023 02:26:40 GMT
P3P: CP="This is not a P3P policy! See g.co/p3phelp for more info."
Content-Security-Policy: require-trusted-types-for 'script';report-uri https://csp.withgoogle.com/csp/docs-tt
Referrer-Policy: origin
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-XSS-Protection: 1; mode=block
Server: GSE
Set-Cookie: NID=511=ToAg9ZOnUrPP99W1c78GH614xiESLaG9rUk4fEbRI8Jh2BhhckU1_ZJdqZzM3XgB91-QwCe1e8n3tWCBDOuucpcGs14etStFxp_2IK2thIWosYF0wRlXfUpZFufSRVcSQRyaQFoap26xDcHTtfbsZgO7EdB_XDyiixLBhk2B8wY; expires=Sat, 20-Apr-2024 02:26:39 GMT; path=/; domain=.google.com; HttpOnly
Alt-Svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
Accept-Ranges: none
Vary: Sec-Fetch-Dest, Sec-Fetch-Mode, Sec-Fetch-Site,Accept-Encoding
Connection: close
Transfer-Encoding: chunked

 ]
[2023-10-20 07:56:41,041: ERROR: main: File is not a zip file ]
Traceback (most recent call last):
  File "E:\Kidney-Disease-Classification-DL\main.py", line 9, in <module>
    data_ingestion.main()
  File "E:\Kidney-Disease-Classification-DL\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 16, in main
    data_ingestion.extract_zip_file()
  File "e:\chicken-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 30, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\ProgramData\anaconda3\Lib\zipfile.py", line 1302, in __init__
    self._RealGetContents()
  File "C:\ProgramData\anaconda3\Lib\zipfile.py", line 1369, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
